### PR TITLE
feat(pictograms): adds pictos and updates categories

### PR DESCRIPTION
**Pull request by chrisconnors-ibm**
#### Changelog

**New**

- mexico--city--angel--of--independence.svg
- mexico--city--museo--soumaya.svg
- milan--duomo--di--milano.svg
- milan--skyscrapers.svg

**Changed**

- categories.yml
- pictogram-master.ai
- metadata.yml

**reviewers**

verify svgs exist/work. search against aliases (`museum`and `statue`). all four grouped within `Cities`.

### CODE REVIEW

File: packages/pictograms/categories.yml [hunk 1]
```diff
@@ -61,6 +61,10 @@ categories:
     - madrid--skyscrapers
     - madrid--statue
     - melbourne
+    - mexico--city--angel--of--independence
+    - mexico--city--museo--soumaya
+    - milan--duomo--di--milano
+    - milan--skyscrapers.svg
```
Comment (jnm2377) [inline packages/pictograms/categories.yml]: Is there a reason this one includes `.svg` and the others don't?

**Comment by jnm2377** [comment]
Also can we update the commit title to use proper commit syntax? Thanks!

File: packages/pictograms/categories.yml (hunk 1 above)
Comment (joshblack) [inline packages/pictograms/categories.yml]:
```suggestion
    - milan--skyscrapers
```

per @jnm2377 comments 👍

**Review by joshblack — CHANGES_REQUESTED** [review]

**Review by jnm2377 — APPROVED** [review]

**Comment by jnm2377** [comment]
hmm... I think the pictograms need to be added to testing maybe? It seems like ci is failing bc it says they're missing.

**Review by joshblack — APPROVED** [review]

**Review by joshblack — CHANGES_REQUESTED** [review]
Looks like CI is noting that they need to be defined in metadata.yml

**Comment by joshblack** [comment]
Bump @chrisconnors-ibm do you have a sec to add in the metadata into `metadata.yml`?

**Review by joshblack — APPROVED** [review]

**Pull request merged**